app/components/pools/pools-controller.ts

```diff
@@ -34,17 +34,18 @@ export class PoolController {
 		}
 	}
 
-	reloadData() {
+	reloadData = () => {
```

Comment:
reloadData and updateData should be like normal functions. the arrow should be moved to where they are getting called. You can refer dashboard for more info.